Gemfile

```diff
@@ -113,7 +113,6 @@ gem 'logstop'
 gem 'yabeda-rails'
 gem 'yabeda-prometheus'
 gem 'yabeda-puma-plugin'
-gem 'yabeda-graphql'
```

Comment:
Is this gem no longer helpful?

Reply:
@eanders My understanding is that this gem doesn't do anything unless it is enabled so I think it is currently a no-op. (according to the docs, you have to add `use Yabeda::GraphQL` to your schema).

We could enable it but I'm not sure this gem is something we should be using in production (it's not widely used, pre-1.0, written targeting an older version or ruby graphql).